docs/webstorm-run-configurations.md

```
@@ -0,0 +1,111 @@
+2. Fill in:
+   - **Name:** e.g. `Desktop: build:unpack`
+   - **package.json:** pick the right workspace's `package.json` (e.g. `apps/desktop/package.json`)
+   - **Command:** `run`
+   - **Scripts:** the script name (e.g. `build:unpack`)
+   - **Package manager:** `pnpm` (or `project` if pnpm is the project default)
+   - **Node interpreter:** `project` (inherits from project-level Node settings)
+3. Apply, close, run.
+
+For scripts that need CLI arguments (e.g. `smoke:packaged-opencode`), add them to the **Arguments** field with `--` as the separator:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)